includes/Core/Conversion_Tracking/Conversion_Tracking.php

```diff
@@ -92,6 +92,10 @@ public function register() {
 		$this->conversion_tracking_settings->register();
 		$this->rest_conversion_tracking_controller->register();
 
+		if ( ! $this->conversion_tracking_settings->is_conversion_tracking_enabled() ) {
+			return;
```

Comment:
Let's move this check inside the `wp_enqueue_scripts` hook, probably above the `if ( ! did_action( 'googlesitekit_ads_init_tag' ) && ! did_action( 'googlesitekit_analytics-4_init_tag' ) ) {` conditional check, since this is the earliest point we decide if any other code will execute

Reply:
Thanks @zutigrm this has been addressed in my latest commit.